interface1.js

```diff
@@ -884,35 +884,200 @@ window.addEventListener('load', function(){
+							if (isNaN(spotForm.genStringsMax[i].value)){
+								maxProblem = 1;
+							}
+						}
+					}else{
+						if (isNaN(spotForm.genStringsMin.value)){
+							minProblem = 1;
+						}
+						if (isNaN(spotForm.genStringsMax.value)){
+							maxProblem = 1;
+						}
+					}
+				}
+				if (minProblem === 0 && maxProblem === 0){
+					/* checking if the input value is more than 0 */
+					for(var i=0; i<length; i++){
+						if (length > 1){
+							if (spotForm.genStringsInput[i].value !== ""){
+								if (Number(spotForm.genStringsMin[i].value) <= 0){
+									minProblem = 1;
+								}
+								if (Number(spotForm.genStringsMax[i].value) <= 0){
+									maxProblem = 1;
+								}
+							}
+						}else{
+							if (Number(spotForm.genStringsMin.value) <= 0){
+								minProblem = 1;
+							}
+							if (Number(spotForm.genStringsMax.value) <= 0){
+								maxProblem = 1;
+							}
+						}
+					}
+					if (minProblem === 0 && maxProblem === 0){
+						/* checking if the input value is more than 10*/
+						for(var i=0; i<length; i++){
+							if (length > 1){
+								if (spotForm.genStringsInput[i].value !== ""){
+									if (Number(spotForm.genStringsMin[i].value) >= 10){
+										minProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax[i].value) >= 10){
+										maxProblem = 1;
+									}
+								}
+							}else{
+								if (Number(spotForm.genStringsMin.value) >= 10){
+									minProblem = 1;
+								}
+								if (Number(spotForm.genStringsMax.value) >= 10){
+									maxProblem = 1;
+								}
+							}
+						}
+						var minmax = 0; //flag checker for min value being less than max value
+						if (minProblem === 0 && maxProblem === 0){
+							/* checking if the input is more than 5 and if min value is less than max value */
+							for(var i=0; i<length; i++){
+								if (length > 1){
+									if (spotForm.genStringsInput[i].value !== ""){
+										if (Number(spotForm.genStringsMin[i].value) >= 5){
+											minProblem = 1;
+										}
+										if (Number(spotForm.genStringsMax[i].value) >= 5){
+											maxProblem = 1;
+										}
+										if (Number(spotForm.genStringsMax[i].value) <  Number(spotForm.genStringsMin[i].value)){
+											minmax = 1;
+										}
+									}
+								}else{
+									if (Number(spotForm.genStringsMin.value) >= 5){
+										minProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax.value) >= 5){
+										maxProblem = 1;
+									}
+									if (Number(spotForm.genStringsMax.value) <  Number(spotForm.genStringsMin.value)){
+										minmax = 1;
+									}
+								}
+							}
+							if (minmax === 0){
+								/* display warning if the input value is more than 5 */
+								if (minProblem === 1 || maxProblem === 1){
+									minProblem = 0;
+									maxProblem = 0;
+									displayWarning("Min or Max input is greater than or equal to 5 which may cause your browser to freeze due to too many terminal strings being generated.");
```

Comment:
This warning is displaying too late. The calculation of terminal strings runs before the warning appears, so the browser seems to just become unresponsive with no warning. Can you actually convert it to a confirm() dialog, so the user has a chance to change their minds before they freeze / lag their browser? You may also need to move it to a different spot in the logic to get it to appear in time.

Reply:
I will switched it to confirm statement.